### ISSUE TITLE
I tried your code in MatLAB. But can't able to execute 

### ISSUE THREAD

**Issue by gopi1989** [jpadolf/Pedestrian-Detection-Code#1]
Please Kindly update the dependency files for this MATLAB Project:
1. Need Dependency Modal files.
2. Instruction document for execution

**Comment by jpadolf** [comment]
Hello,
I'm sorry for this issue! We were still finishing up the file dependencies necessary for the code to be useable for others. We have now included a README.txt that should include instructions on how to properly execute the code as well as where to find the classification model. Let me know if you have any more questions.

**Comment by gopi1989** [comment]
Yes , Sure will check the Readme.txt , Let you know the Build status . Thanks for your valuable reply @jpadolf

**Comment by gopi1989** [comment]
Hi , 

I tried the code by followed the read me instructions. I am struggling with one warning in "mod2" .
I am struggled to resolve the warning. will you please more specific about the mod2 and where i have to use it

Thanks,

Regards,
Gopi Krishnan.R

**Comment by jpadolf** [comment]
Hi Gopi,
So "mod2" is the Classification model variable saved in the Matlab workspace "model.m". This is a very, very large file (~200MB), so I did not include it in the Github repository, but instead included it in the Dropbox directory that is mentioned in the README. I suggest downloading the files from Github and Dropbox and saving them in the following directory structure:
![directorystructure](https://cloud.githubusercontent.com/assets/6997313/7595136/cdbd700c-f8aa-11e4-9dff-05c68a7151bd.png)
You should then change the working directory of MATLAB to your Code directory (the Github repository) and run the startup.m script to load the model. Let me know if you have any more errors!
- Jeff

**Comment by gopi1989** [comment]
Hi jeff,

```
     Thanks, will change the directory of the model.mat and execute the code and update it to you  
```

**Comment by gopi1989** [comment]
Yes Jeff, Thanks for your Support ,I executed successfully in Linux platform. I tries to port in windows but facing an issues. Have you tries in windows

**Comment by jpadolf** [comment]
Hey Gopi,
We did test this extensively in Windows, and everything should be the same there. Make sure that you have the correct MATLAB Toolboxes installed on your Windows machine. I know that the "extractHOGFeatures" function we use requires the Computer Vision System Toolbox. However, if its working on the Linix platform, this probably isn't the problem.

Can I ask what specific error you're getting?
- Jeff

**Comment by gopi1989** [comment]
Yes Jeff,

```
   I send the error log to you ,when i tries in windows platform.
```

Error Log:

Warning: Variable 'mod2' originally saved as a ClassificationSVM cannot be
instantiated as an object and will be read in as a uint32. 

> In startup at 10

**Comment by jpadolf** [comment]
Hi Gopi,

The error is telling you that your version of MATLAB does not understand the variable type 'ClassificationSVM'. This means that you most likely do not have the Statistics and Machine Learning Toolbox required to read this variable. If you'd like to check what Toolboxes you have installed on your machine, type "ver" into the command line. Best of luck!
- Jeff

**Comment by gopi1989** [comment]
Yes Jeff,
 Thanks for your reply .

**Comment by zhangxiaotao2015** [comment]
Hi Jeff,
  I'm from China. I can't get your files from dropbox,could you give me an email for it.Thanks!My email is 1162737802@qq.com

**Comment by leonkding** [comment]
Hi Jeff,
I have the same problem, I can't get your files from dropbox either. So can you reply with a new link or send me via email? My email is leondg@outlook.com. Thank you.

**Comment by StefaniaTp** [comment]
Hi Jeff, 
I have problems with the Dropbox link. Can you send me the files? You can send me another link or an email to stefania.trapani@studenti.polito.it (the email is the same for both cases). 
Thank you in advance.

**Comment by Sabrinehamdi** [comment]
Hi Mr,
I have problem with Dropbox link. I can't download  'PdestrianDetection/Files' so our pedestrian detection code can't also runned. please can you send me another link to download Files. Thank you  very much

**Comment by Sabrinehamdi** [comment]
Please can you sent me the files in this sabrinehs@hotmail.fr

**Comment by litalin** [comment]
Hello Jeff,
I have a problem with my Dropbox link. Can you send me the documents? You can send me another link or email to aleetalin@gmail.com.it (the email is the same in both cases).
Thank you in advance.
